pkg/analysis/passes/gomanifest/gomanifest.go

```diff
@@ -35,7 +35,7 @@ var Analyzer = &analysis.Analyzer{
 func run(pass *analysis.Pass) (interface{}, error) {
 	metadataBody, ok := pass.ResultOf[metadata.Analyzer].([]byte)
 	if !ok {
-		return nil, errors.New("metadata not found")
+		return nil, nil
```

Comment:
This change is intended. We don't want to fail the process with this. Instead we should simply skip the validator. the `metadata ` validator is already responsible of reporting this error so this is redundant logic.